### PR TITLE
Averaging to adding

### PR DESCRIPTION
**Pull request by JennaGustafson**

### CODE REVIEW

File: execute_process_mass_data.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
 # plot_binned_pmf_array(gases, list_of_arrays, create_bins_results, bin_compositions_results)
 # save_array_pmf_data(gases, list_of_arrays, create_bins_results, bin_compositions_results)
 kl_divergence = information_gain(gases, list_of_arrays, bin_compositions_results, create_bins_results)
-combined_kld, ordered_by_gas, best_arrays, ordered_kld_w_array, average_kld = choose_best_arrays(gases, kl_divergence)
+# combined_kld, ordered_by_gas, best_arrays, ordered_kld_w_array, average_kld = choose_best_arrays(gases, kl_divergence)
```
Comment (paulboone) [inline execute_process_mass_data.py]: commented out code

File: execute_process_mass_data.py [hunk 2]
```diff
@@ -49,7 +49,7 @@
 # plot_binned_pmf_array(gases, list_of_arrays, create_bins_results, bin_compositions_results)
```
Comment (paulboone) [inline execute_process_mass_data.py]: generally the word "squash" in a commit message indicates that you forgot to actually squash

File: execute_process_mass_data.py [hunk 3]
```diff
@@ -41,28 +41,28 @@
...
-print("\nThe MOF array with the lowest informaion content, of %s, is: %s \n" % (str(ordered_kld_w_array[len(ordered_kld_w_array)-1]['KLD']),
-                                                                                str(ordered_kld_w_array[len(ordered_kld_w_array)-1]['mof array'])))
-
-for result in best_arrays:
-    print("The best array for %s consists of MOFs: %s" % (result['gas'], result['mof array']))
-print('\nThe average Kullback-Liebler Divergence for a: ')
-for num_mofs in range(0,len(average_kld)):
-    print('\t%s MOF array is %s' %(num_mofs + 1, average_kld[num_mofs]))
+# write_output_data('saved_results/ordered_by_kld_product_%s.csv' % (datetime.now().strftime("%Y_%m_%d__%H_%M_%S")), combined_kld)
+# write_output_data('saved_results/ordered_by_gas_kld_%s.csv' % (datetime.now().strftime("%Y_%m_%d__%H_%M_%S")), ordered_by_gas)
```
Comment (paulboone) [inline execute_process_mass_data.py]: commented out code

**Review by paulboone — CHANGES_REQUESTED** [review]
Looks good. You had some commenting in / out of code which should never really be committed. Can you check those and make sure they were actually removed?

**Comment by JennaGustafson** [comment]
There were two lines of code commented out in the final version of process_mass_data.py, so I deleted them (they are not used) and squashed that commit further back where I initially committed the commented out code. The final version looks better.

**Review by paulboone — APPROVED** [review]

**Pull request merged**